client/src/components/FilterAccordion/FilterAccordion.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { styled, Accordion, AccordionSummary, AccordionDetails, Typography } from '@mui/material';
+
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+
+const StyledAccordion = styled((props) => <Accordion elevation={0} square defaultExpanded {...props} />)(
+  ({ theme }) => ({
+    position: 'relative',
+    boxShadow: 0,
+
+    '&:after': {
+      content: '""',
+      position: 'absolute',
+      bottom: 0,
+      left: 0,
+      height: '1px',
+      width: '100%',
+      borderTop: `1px solid ${theme.palette.divider}`,
+    },
+
+    '&:before': {
+      background: 'none',
+    }
+
+  })
+);
+
+const StyledAccordionDetails = styled(AccordionDetails)({
+  padding: '0 0 40px',
+});
+
+const AccordionTitle = styled((props) => <Typography variant="body1" {...props} />)(() => ({
```

Comment:
`Typography` has "body1" variant by default
```suggestion
const AccordionTitle = styled(Typography)(() => ({
```

Reply:
Done